### ISSUE TITLE
error: RPC failed; curl 18 transfer

### ISSUE THREAD

**Issue by jc5706** [SonOfLysander/git-openssl-shellscript#18]
Tried on a fresh install of Ubuntu 18.04.6 on VirtualBox 6.1 on Windows 10

Build end, puts git version  2.38.0-rc1 in /usr/bin/git

But when I try to clone a repo, it says:

remote: Enumerating objects: 26528, done.
remote: Counting objects: 100% (150/150), done.
remote: Compressing objects: 100% (136/136), done.
**error: RPC failed; curl 18 transfer closed with outstanding read data remaining**
error: 6687 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

**Comment by nikole-dunixi** [comment]
I'm not able to replicate this locally. The problem you're having looks to be rooted in your internet connection. I'd double check your local configuration as well.

Googling your error seems to point to this as the culprit: https://stackoverflow.com/a/73079738/1478636

**Comment by jc5706** [comment]
Appreciate your quick answer. It certainly makes sense what you say as it´s hard to believe git was that much broken. But I have tried many different possibilities, mostly with fresh ubuntus on VMs. 
I have a good internet and even if it was its fault I would have noticed the problem being intermitent. I also have no antivirus, not even the W defender and the firewall is off. It's a developer machine, there's nothing of permanent value (I mean, the physical win machine).
From the recipes in your links, some of them were old stuff to me, others new, I tried them to no avail...

I know there are workarounds, like shallow clones, etc, but it's a makefile that uses git, so these don't apply.

I ran out of ideas (on the git side). I'm thinking now of blaming either the machine or its network settings or dive into that makefile and make all git clone shallow...

Thanks again for your help
